src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/Models/PSContainerRegistryWebhookEvent.cs

```diff
@@ -0,0 +1,32 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.ContainerRegistry.Models;
+
+namespace Microsoft.Azure.Commands.ContainerRegistry
+{
+    public class PSContainerRegistryWebhookEvent
+    {
+        public PSContainerRegistryWebhookEvent(EventModel webhookEvent)
+        {
+            Id = webhookEvent?.Id;
+            EventRequestMessage = webhookEvent?.EventRequestMessage;
+            EventResponseMessage = webhookEvent?.EventResponseMessage;
+        }
+
+        public string Id { get; set; }
+        public EventRequestMessage EventRequestMessage { get; set; }
+        public EventResponseMessage EventResponseMessage { get; set; }
```

Comment:
Flatten? The expected output here should be request/response of the event and especially the json request payload. How does Powershell print json? /cc @cormacpayne

Reply:
same as CallbackConfig.